### DISCUSSION TITLE
Add an option to continue playing a map after failing in singleplayer like it's implemented in multiplayer

### DISCUSSION THREAD

**Discussion by Newbie1st** [ppy/osu#31027] [Ideas] [answered]
After forgetting to remove NF several times, I started creating closed solo lobbies whenever I want to play a map I know I potentially won't be able to pass to make sure this never happens again. This option would make my life so much easier.

**Comment by N0Y0U2114** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/discussions/28381
